src/apps/components/CustomAppTokens/CustomAppTokens.tsx

```diff
@@ -80,6 +79,7 @@ const CustomAppTokens: React.FC<CustomAppTokensProps> = props => {
                 </TableCell>
                 <TableCell className={classes.colActions}>
                   <IconButton
+                    variant="secondary"
                     color="primary"
```

Comment:
What's the default color for `IconButton`? If it's also "primary" then we should skip it like we do in Buttons

Reply:
It's `default`, they become `primary` on hover